src/content/docs/usage/workspaces.mdx

```diff
@@ -92,7 +122,7 @@ daytona   | Project daytona started
 
 ### From an Arbitrary Git URL
 
-Creating a Workspace from an arbitrary Git URL allows you to set up a new environment by directly specifying the URL of the remote repository. This method is useful for setting up environments that are not linked to a pre-configured Git Provider in Daytona.
+Creating a Workspace from an arbitrary Git URL sets up a new development environment by directly specifying the URL of the remote repository. This method is used to set up environments that are not linked to a pre-configured Git Provider in Daytona.
```

Comment:
> This method is used to set up environments that are not linked to a pre-configured Git Provider in Daytona.

I suggest removing this. It's not entirely true since it can also be used for repos that are linked to pre-configured Git Providers. Maybe users don't want to go through the TUI fully